README.md

```diff
@@ -51,7 +51,7 @@ When clicking submit, every recipe containing the word will be updated (tags are
 2. Choose one of the included configurations [here](https://github.com/vabene1111/recipes/tree/develop/docs/docker).
 3. Copy (if needed) the `docker-compose.override.yml.template` to `docker-compose.override.yml` and uncomment the configurations you need.
 4. Start the container (`docker-compose up -d`)
-5. Create a default user by executing into the container with `docker-compose exec web_recipes sh` and run `python3 manage.py createsuperuser`.
+5. Create a default user by running `docker-compose exec -it <web_recipes> createsuperuser`. **Replace "<web_recipes>" with your instance name which can be found via running `docker ps`**
```

Comment:
Because of using a venv I simplified the superuser creation. Otherwise you would have to run `source venv/bin/activate` before executing `python manage.py createsuperuser`

Reply:
why use venv in a dockerized environment ? there is only one host running in the container anyway. since docker compose names the containers i dont think the replacing part is necessary (see update.sh)

Reply:
It is a cleaner more bugfree approach.
https://vsupalov.com/virtualenv-in-docker/

Reply:
ok it appears to have some benefits. even though i find them rather slim. there is not much complexity added so i guess its fine.